discovery-service/src/test/java/org/zowe/apiml/discovery/GatewayNotifierTest.java

```diff
@@ -134,4 +144,96 @@ public void testNotificationFailed() {
         );
     }
 
+    @Test
+    public void testDistributeInvalidatedCredentials() {
+        InstanceInfo targetInstanceInfo = createInstanceInfo("host", 1000, 1433);
+        String targetInstanceId = targetInstanceInfo.getInstanceId();
+
+        InstanceInfo gatewayInstance = createInstanceInfo("gateway", 111, 123);
+        String gatewayUrl = "https://gateway:123/auth/distribute/" + targetInstanceId;
+
+        RestTemplate restTemplate = mock(RestTemplate.class);
+        MessageService messageService = mock(MessageService.class);
+        GatewayNotifier gatewayNotifier = new GatewayNotifierSync(restTemplate, messageService);
```

Comment:
I think you can move 
`RestTemplate restTemplate = mock(RestTemplate.class);
        MessageService messageService = mock(MessageService.class);
        GatewayNotifier gatewayNotifier = new GatewayNotifierSync(restTemplate, messageService);`
to the `setup` method since they are used by severals tests. Just for better readability of the tests

Reply:
done